install_config/configuring_aws.adoc

```diff
@@ -12,10 +12,11 @@ toc::[]
 
 == Overview
 {product-title} can be configured to access an
-link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/concepts.html[AWS EC2 infrastructure], including
-xref:../install_config/persistent_storage/persistent_storage_aws.adoc#install-config-persistent-storage-persistent-storage-aws[using AWS
-volumes as persistent storage] for application data. After AWS is configured
-properly, some additional configurations will need to be completed on the
+link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/concepts.html[AWS EC2
+infrastructure], including
+xref:../install_config/persistent_storage/persistent_storage_aws.adoc#install-config-persistent-storage-persistent-storage-aws[using
+AWS volumes as persistent storage] for application data. After AWS is configured
```

Comment:
Maybe s/After AWS is configured properly, some additional configurations must be completed on the {product-title} hosts./After you configure AWS, you must modify the configuration of the {product-title} hosts.